MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/javaservices/MatrixApiActivity.java

```diff
@@ -174,7 +175,7 @@ public void onFailure(Call<DirectionsMatrixResponse> call, Throwable throwable)
   private void addMarkers() {
     Icon lightningBoltIcon = IconFactory.getInstance(MatrixApiActivity.this)
       .fromResource(R.drawable.lightning_bolt);
-    for (Feature feature : featureCollection.getFeatures()) {
+    for (com.mapbox.geojson.Feature feature : featureCollection.features()) {
```

Comment:
Any reason this is using the full namespace here?

Reply:
Fixed. Full namespace removed